font/Cargo.toml

```
@@ -24,3 +27,4 @@ core-foundation-sys = "0.6"
 [target.'cfg(windows)'.dependencies]
 font-loader = "0.6.0"
 rusttype = "0.4.1"
+dwrote = { version = "0.8.0", optional = true }
```

```suggestion
dwrote = { git = "https://github.com/servo/dwrote-rs.git", optional = true }
```
Two weeks ago, the `dwrote` fork for servo added support for font fallback.

---

Please note that the `servo-dwrote` fork would probably be a better idea than a git dependencies.

---

@chrisduerr Unfortunately, the `servo-dwrote` crate, while it links to the right github repository, was last updated 2 years ago, so it's wayyy behind.

---

Then the way to go would probably be to ask them to push a new version.

---

`servo-dwrote` was a temporary fork that is no longer used. `dwrote` is the up to date crate.